### PR TITLE
Create datamgmt.md

### PR DESCRIPTION
**Pull request by nehamoopen**
Comments on data management

### CODE REVIEW

File: datamgmt.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Data storage and management
+
+This task requires some amount of work before data collection, although we tend to work on it during data collection instead. This task could involve the following:
```
Comment (chartgerink) [inline datamgmt.md:3]: This is a good first sentence! :raised_hand_with_fingers_splayed:

Comment (chartgerink) [reply datamgmt.md:3]: @piatio could you take a look at this?

**Review by chartgerink — CHANGES_REQUESTED** [review]
Good first start needs more work.

File: datamgmt.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+1. Writing a data management plan (DMP)
+2. Knowing where you're going to store your data as it comes in, keeping in mind the security and privacy requirements
+3. Decide on your method for version control and communicate to colleagues/supervisors
```
Comment (K-Hoyer) [inline datamgmt.md:6]: Most department also have specific rules, need to check those

**Comment by PiaTio** [comment]
Hoooooi!

2018-03-09 11:50 GMT+01:00 nmoopen <notifications@github.com>:

**Comment by K-Hoyer** [comment]
Helllo hello☺

Karlijn Hoyer

PhD candidate – Department of Social Psychology – Tilburg School of Social and Behavioral Sciences - Tilburg University | Postbus 90153, 5000 LE Tilburg | Simon building, room S414 | +31 (0) 13 466 3224

[cid:image006.png@01D2E8DF.FDD39A60]<http://www.tilburguniversity.edu/90years>

**Pull request merged**